packages/monitoring-experience-api/src/middleware/ipWhitelistValidation.ts

```diff
@@ -0,0 +1,42 @@
+import type {
+  AbstractController,
+  UnknownRequest
+} from '@verkkokauppa/core'
+import ipRangeCheck from 'ip-range-check'
+import { IpAddressValidationError } from '../errors'
+
+export const withIpWhitelistValidation = <
+    // eslint-disable-next-line prettier/prettier
+    TController extends abstract new (...args: any[]) => AbstractController
+    >(
+    Controller: TController
+): TController => {
+  abstract class IpAddressWhitelistValidationController extends Controller {
+    protected async validateRequest(req: UnknownRequest) {
+      const validIps = process.env.ROUTE_PUBLIC_MONITORING_IP_WHITELIST
+          ? process.env.ROUTE_PUBLIC_MONITORING_IP_WHITELIST.split(' ')
```

Comment:
Should the split character be comma?

Reply:
No, the IP:s are in the environment variable separated only with one space.